translations/de/Info/task_description.html

```diff
@@ -2,7 +2,7 @@
      style="width: 300px;float: right; margin-top: 30px; padding: 0 10px 0 30px;"/>
 
 <p>
-    Erinnerst du dich an<a href="https://de.wikipedia.org/wiki/Stellenwertsystem#Basis"> die Basiszahl</a> und
+    Erinnerst du dich an <a href="https://de.wikipedia.org/wiki/Stellenwertsystem#Basis">die Basiszahl</a> und 
```

Comment:
Should read as: Erinnerst **Du Dich** an...

I recommend to use [Rechtschreibprüfung Online](http://www.duden.de/rechtschreibpruefung-online/) by Duden.